android/build.gradle

```diff
@@ -144,26 +140,39 @@ repositories {
     mavenCentral()
 }
 
-def kotlin_version = getExtOrDefault('kotlinVersion', project.properties['RNSAC_kotlinVersion'])
+def media3_version = safeExtGet('media3Version', '1.1.1')
```

Comment:
should be moved to `gradle.properties` (see #3334)

Reply:
everythings are moved, please check again